apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -48,6 +48,7 @@
 
 -define(NAME_RE, "^[A-Za-z]+[A-Za-z0-9-_.]*$").
 -define(TAGS, [<<"Plugins">>]).
+-define(VSN_WILDCARD, "-*[0-9]*.[0-9]*.[0-9]*.tar.gz").
```

Comment:
My motivation was to match versions like `-v5.0.0.`, but perhaps we shouldn't expect, and can use `-` indeed..

Reply:
`-v5.0.0` is not a valid semver (see: https://semver.org/).
I think it's worth matching `-` instead of `-*`, as `-*` also matches: `plugin-a-<vsn>.tar.gz`, for example.
We can also make it more permissive to allow versions like 1.0, or even 1:
```
24> filelib:wildcard("plugin" ++ "-[0-9]*.tar.gz").
["plugin-1-5.0.0.tar.gz","plugin-5-rc3.tar.gz",
 "plugin-5.0.0-rc.3.tar.gz","plugin-5.0.0.tar.gz",
 "plugin-5.0.tar.gz","plugin-5000.tar.gz"]
```
I guess any more solid solution would require enforcing some name restrictions and version requirements, which can be done as a separate improvement. It may involve disallow using '-' in the name part and/or validate version part to comply with semver or some other well defined pattern.

Reply:
After discussion with @zmstone , I've changed it to simple `"-*.tar.gz"` wildcard. 
It follows OTP naming convention `<app_name>-<vsn>`, where a version can be anything and app name is snake case (no `-` allowed).